sdk/purview/azure-analytics-purview-workflow/src/test/java/com/azure/analytics/purview/workflow/PurviewWorkflowClientTestBase.java

```diff
@@ -37,7 +36,14 @@ protected void beforeTest() {
         } else {
             purviewWorkflowClientbuilder
                 .httpClient(HttpClient.createDefault())
-                .credential(new DefaultAzureCredentialBuilder().build());
+                .credential(new UsernamePasswordCredentialBuilder()
+                    .clientId(Configuration.getGlobalConfiguration().get("CLIENTID", "clientId"))
```

Comment:
Should these be named AZURE_CLIENT_ID etc..

Reply:
It is what the SDK owner uses. I would stay with what it is until they change.